src/Sylius/Bundle/ApiBundle/test/tests/FooSyliusResourceTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\test\tests;
```

Comment:
This namespace shows, that something is odd with this config. Can we do something like this
`Sylius\Bundle\ApiBundle\test\` <- main folder for test app
`Sylius\Bundle\ApiBundle\test\Application` <- main folder for test app
`Sylius\Bundle\ApiBundle\test\Application\config` <- main folder for test app 
`Sylius\Bundle\ApiBundle\test\Application\src` <- main folder for test code
`Sylius\Bundle\ApiBundle\test\Tests` <- main folder for tests classes